server/services/licenceSearchService.js

```diff
@@ -94,7 +99,7 @@ module.exports = function createLicenceSearchService(
           prisonerFirstname: prisoner.firstName,
           prisonLastname: prisoner.lastName,
           HDCED: moment(prisoner.homeDetentionCurfewEligibilityDate).format('DD-MM-YYYY'),
-          PDU: probationDetail.offenderManagers[0].probationArea.description,
+          PDU: probationDetail ? probationDetail.offenderManagers[0].probationArea.description : '',
```

Comment:
Should we look for an active offender manager? 
might be better to do `?.[0]` and then `?.` to handle no offender managers rather than blow up?

Reply:
optional chaining causing issues with linting in a javascript file so opted for using destructuring to get description value.

Reply:
Also added a look up for the active community offender manager in this function 😅